engine.go

```diff
@@ -545,8 +548,9 @@ func (t *T) shouldLog() bool {
 	return t.rawLog != nil || t.tbLog
 }
 
-// Context returns a context.Context associated with the test.
-// It is valid only for the duration of the rapid check.
+// Context returns a context.Context associated with the property check.
+// It is valid only for the duration of the check,
```

Comment:
Wording here is ambiguous: `for the duration of the check` can be interpreted as "during the Check()".